### DISCUSSION TITLE
Large indent in tag 'body'

### DISCUSSION THREAD

**Discussion by AHTOHO4** [igordanchenko/yet-another-react-lightbox#320] [General]
Hi.
I have encountered the following problem:
When I open the likebox, I see that the body tag is assigned the class yarl__no_scroll, and so is padding-right. But the problem is that when I open devtools, I see that the padding is very large. The more devtools is opened (i.e. technically, the smaller the window is in relation to the entire screen), the larger the padding-right is assigned

![image](https://github.com/user-attachments/assets/c725295c-324c-46bc-bb4c-3f65d082cd80)
![image](https://github.com/user-attachments/assets/34712c0c-9ea0-40f1-989d-63a4d5bc0bb8)

Unfortunately, I was unable to prepare an example and reproduce the problem in the sandbox. I assume that this is due to the fact that an iframe is used there.

If I first open the lightbox, and then devtools, I see an adequate padding-right: 15px
![image](https://github.com/user-attachments/assets/53e0e34c-37e5-4606-a5fd-45d5a25ae896)

Questions:
1) What could possibly be the problem and how can I try to fix it?
2) Maybe I should somehow remove padding-right, leaving the classes. Will this be the right solution or is it more like a crutch?

**Comment by igordanchenko** [comment]
1) What browser / OS are you using? I've been unable to reproduce in Chrome on Mac and Windows.
2) Are you able to reproduce this issue with the official [examples](https://yet-another-react-lightbox.com/examples/basic) on the documentation website?
3) Are you able to reproduce this on a different desktop / laptop / browser?
4) Have you tried reproducing in incognito mode?
5) Have you tried disabling all browser plugins?

> What could possibly be the problem and how can I try to fix it?

I'm not sure, but it appears that for some reason your browser incorrectly reports its window inner width that includes the dev tools pane.

> Maybe I should somehow remove padding-right, leaving the classes. Will this be the right solution or is it more like a crutch?

It doesn't sound like the right solution, but you can disable this feature with the `noScroll={{ disabled: true }}`.

**Reply by AHTOHO4** [reply]
I can't reproduce the bug anymore, maybe the problem was on my side. If I find something, I'll definitely let you know.
Thanks!
